test/pick.js

```diff
@@ -4,12 +4,16 @@ var R = require('..');
 
 
 describe('pick', function() {
-  var obj = {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6};
+  var obj = {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6, 1: 7};
```

Comment:
I believe that property names which are not valid identifiers must be quoted in old environments. Perhaps we should do so here.